app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
+
+  def three_most_recent_post
+    posts.last(3)
+  end
```

Comment:
- [ ] Great job adding into the User model a method that returns 3 most recent posts 👍🏻 However I can suggest using this approach which is more better than using the last method👇🏻 

```ruby
def recent_post
  posts.order('created_at Desc').limit(3)
end
```